lapack/gonum/dpbtf2.go

```diff
@@ -89,16 +91,16 @@ func (Implementation) Dpbtf2(ul blas.Uplo, n, kd int, ab []float64, ldab int) (o
 		}
 		return true
 	}
+	// Compute the Cholesky factorization A = L * L^T.
 	for j := 0; j < n; j++ {
-		// Compute L(J,J) and test for non positive-definiteness.
+		// Compute L(j,j) and test for non-positive-definiteness.
```

Comment:
We don't conventionally write hyphenation like this. I think we are wrong, so please leave it as is. (Just by way of a note).

Reply:
Ok. The changes in this file are unnecessary but since I was going through it anyway, I thought I could do some minimal cleanup.

Reply:
Sorry. That was really unclear. Leave it how you have it.